resources/Microsoft.DotNet.Dsc/Microsoft.DotNet.Dsc.psm1

```diff
@@ -5,6 +5,9 @@ $ErrorActionPreference = 'Stop'
 $PSNativeCommandUseErrorActionPreference = $true
 Set-StrictMode -Version Latest
 
+# Set environment variable always to true to skip first run experience
+$env:DOTNET_NOLOGO = $true
```

Comment:
For reference:
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#dotnet_skip_first_time_experience

#Resolved